src/irr/asset/CGLTFWriter.h

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2020 AnastaZIuk
+// This file is part of the "Irrlicht Engine".
+// For conditions of distribution and use, see copyright notice in irrlicht.h
+
+#ifdef _IRR_COMPILE_WITH_GLTF_WRITER_
+
+#include "IrrCompileConfig.h"
+#include "irr/asset/ICPUImageView.h"
+#include "irr/asset/IAssetLoader.h"
+
+namespace irr
+{
+	namespace asset
+	{
+		//! glTF Writer capable of writing .gltf files
+		/*
+			glTF bridges the gap between 3D content creation tools and modern 3D applications
+			by providing an efficient, extensible, interoperable format for the transmission and loading of 3D content.
+		*/
+
+		class CGLTFWriter final : public asset::IAssetWriter
+		{
+			protected:
+				virtual ~CGLTFWriter() {}
+
+			public:
+				CGLTFWriter() {}
+
+				virtual const char** getAssociatedFileExtensions() const override
+				{
+					static const char* extensions[]{ "gltf", nullptr };
+					return extensions;
+				}
+
+				uint64_t getSupportedAssetTypesBitfield() const override { return asset::IAsset::ET_MESH; }
+
+				uint32_t getSupportedFlags() override { return asset::EWF_NONE; }
```

Comment:
binary?

Reply:
at the moment only none, I will change it in future

Reply:
still a TODO